### PR TITLE
Adding pre-commit hooks support for black and flake8(PEP8 compliance …

### PR DESCRIPTION
**Pull request by BodaleDenis**
- I've added pre-commit hooks support for the project
- I've included black and flake8 for compliance check with PEP8
I hope it is OK, it would be nice to test on your side too

P.S: All hooks have passed directly, so that is a good job 😄

### CODE REVIEW

**Comment by BodaleDenis** [comment]
pip install pre-commit
pre-commit install

This should be enough, in order to have pre-commit up and running locally

File: requirements.txt [hunk 1]
```diff
@@ -20,3 +20,5 @@ sqlparse==0.4.1
 tornado==6.1
 whitenoise==5.2.0
 zipp==3.4.1
+pre-commit
+flake8
```
Comment (winny-) [inline requirements.txt]: Let's omit these two additions from `requirements.txt`.  If one installs pre-commit (via their OS's package manager or `pip install --user`), that should be sufficient.  If I'm not mistaken, `pre-commit` automatically pulls in tools needed to run the checks, so we shouldn't need `flake8` in that case.

File: .pre-commit-config.yaml [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+    -   id: end-of-file-fixer
+    -   id: trailing-whitespace
+-   repo: https://github.com/psf/black
+    rev: 19.3b0
+    hooks:
+    -   id: black
+-   repo: https://gitlab.com/pycqa/flake8
+    rev: 3.7.9
+    hooks:
+    - id: flake8
```
Comment (winny-) [inline .pre-commit-config.yaml]: I am a little opinionated about formatting YAMLs.  What do you think?
```suggestion
repos:
  - repo: https://github.com/pre-commit/pre-commit-hooks
    rev: v2.3.0
    hooks:
      - id: check-yaml
      - id: end-of-file-fixer
      - id: trailing-whitespace
  - repo: https://github.com/psf/black
    rev: 19.3b0
    hooks:
      - id: black
  - repo: https://gitlab.com/pycqa/flake8
    rev: 3.7.9
    hooks:
      - id: flake8
```

**Review by winny- — CHANGES_REQUESTED** [review]
Thank you for the pull request!  I have a suggested a tormatting change and wanted your opinion on the `requirements.txt` change?  Maybe that change could be omitted?  What do you think?

**Comment by BodaleDenis** [comment]
Ok I will do the changes

**Comment by winny-** [comment]
Will do.  One outstanding discussion item here https://github.com/winny-/sillypaste/pull/12#discussion_r723775724

**Comment by BodaleDenis** [comment]
flake and pre-commit are now removed from requirements.txt

File: requirements.txt [hunk 3]
```diff
@@ -20,3 +20,4 @@ sqlparse==0.4.1
 tornado==6.1
 whitenoise==5.2.0
 zipp==3.4.1
+
```
Comment (winny-) [inline requirements.txt]: This should remove the blank line.

```suggestion
```

**Review by winny- — COMMENTED** [review]
One final change requested, then I'll go ahead and merge

**Comment by winny-** [comment]
I went ahead and committed that code review suggestion to delete a line.  I'm still learning how to use the code review feature, so hoping I'm doing this right!

Thank you for the pull request :-) !  If you want to make further changes feel free to open an issue first to talk about it, and open a PR.  I'll help ensure the changes have a good chance of being merged in.

**Pull request merged**